src/admin-views/editor/fieldset/price.php

```diff
@@ -34,8 +34,17 @@ class="price tribe-dependent"
 			class="ticket_field ticket_form_right"
 			size="7"
 			value="<?php echo esc_attr( $ticket ? $price : null ); ?>"
+			<?php if ( $provider instanceof Tribe__Tickets__Commerce__PayPal__Main ) {
+				echo 'data-validation-is-greater-than="0"',
+					'data-validation-error="' . esc_attr__( 'PayPal tickets require a nonzero price', 'event-tickets' ) . '"';
+			} ?>
 		/>
-		<p class="description ticket_form_right"><?php esc_html_e( 'Leave blank for free tickets', 'event-tickets' ) ?></p>
+		<p class="description ticket_form_right">
+			<?php
+			echo $provider instanceof Tribe__Tickets__Commerce__PayPal__Main
```

Comment:
Stick this logic on the top and pass a variable, avoid logic inside of HTML blocks if possible.